config/routes/overrides.rb

```diff
@@ -1,6 +1,21 @@
+class WhitelistConstraint
```

Comment:
Ignore me about the Blacklist aspect, the update to the message confused me. However, this would be nice to namesapce still.